### DISCUSSION TITLE
Can we save a custom drumkit in v1.1.1?

### DISCUSSION THREAD

**Discussion by Djard** [hydrogen-music/hydrogen#1487] [Hydrogen on Windows]
I followed a Youtube tutorial on how to create a custom drum kit in v0.9.4, up to the point where the kit is saved. From the "Instruments" drop-down menu, "Save Library" was selected.

In v1.1.1, the "Save Library" option appears to have been removed. In the FAQ, I am pointed to Video 5 for the answer; but where is video 5 found?

I see that all the audio samples are saved in a Data directory with an XML.

Can someone lend me a hand and tell me how to save a custom drum kit that I created?

**Comment by theGreatWhiteShark** [comment]
> I followed a Youtube tutorial on how to create a custom drum kit in v0.9.4, up to the point where the kit is saved. From the "Instruments" drop-down menu, "Save Library" was selected.
>
> In v1.1.1, the "Save Library" option appears to have been removed.

There had been some renaming done. In the "Instrument" menu you now only have option affecting single or multiple instruments within the drumkit. Everything acting on the whole drumkit has been moved to the "Drumkits" menu - including the "Save" and "Save As" option.

I would also suggest to have a look at the [documentation](http://hydrogen-music.org/documentation/manual/manual_en.html). v0.9.4 is more than 12 years old...

**Comment by Djard** [comment]
I finally succeeded. No extension needs to be added when saving the file. Thanks for lending a hand.
